cl/recap/tests.py

```diff
@@ -1855,11 +1855,18 @@ def test_create_from_idb_chunk(self) -> None:
         self.assertEqual(Docket.objects.count(), 3)
 
 
-def mock_bucket_open(message_id, r):
+def mock_bucket_open(message_id, r, read_file=False):
     """This function mocks bucket.open() method in order to call a
     recap.email notification fixture.
     """
     test_dir = Path(settings.INSTALL_ROOT) / "cl" / "recap" / "test_assets"
+
+    if read_file:
+        with open(test_dir / message_id, encoding="utf-8") as file:
+            text = file.read()
+            file.close()
+            return text
```

Comment:
You're using a context handler — it'll close the file itself, no?

Reply:
yeah, removed!